plugins/user-settings/src/components/AuthProviders/ProviderSettingsItem.tsx

```diff
@@ -45,17 +46,21 @@ export const ProviderSettingsItem = ({
 }: Props) => {
   const api = useApi(apiRef);
   const [signedIn, setSignedIn] = useState(false);
+  const [profileEmail, setProfileEmail] = useState('');
 
   useEffect(() => {
     let didCancel = false;
 
-    const subscription = api
-      .sessionState$()
-      .subscribe((sessionState: SessionState) => {
-        if (!didCancel) {
-          setSignedIn(sessionState === SessionState.SignedIn);
-        }
-      });
+    const subscription = api.sessionState$().subscribe(() => {
+      if (!didCancel) {
+        api
+          .getProfile({ optional: false })
```

Comment:
This was meant to be true as per the issue.

Reply:
yeah, not sure why I did that